networking/networking_operators/aws_load_balancer_operator/aws-load-balancer-operator-release-notes.adoc

```diff
@@ -23,10 +23,17 @@ For an overview of the AWS Load Balancer Operator, see xref:../../../networking/
 AWS Load Balancer Operator currently does not support AWS GovCloud.
 ====
 
+[id="aws-load-balancer-operator-release-notes-1.2"]
+== AWS Load Balancer Operator 1.2.0
+
```

Comment:
I don't know why https://issues.redhat.com/browse/CFE-1133 was private and hence not added to the advisory. But we should mention the fact that the platform tags (ones defined in `Infrastructure` resource) will be added to all AWS objects created by the controller now.

Reply:
@chiragkyal : is there a reason not to mention this in the release notes?

Reply:
cc @TrilokGeer

Reply:
> @chiragkyal : is there a reason not to mention this in the release notes?

I don't see any strong reason for not mentioning the feature information in the release notes. 

> I don't know why https://issues.redhat.com/browse/CFE-1133 was private and hence not added to the advisory.

@TrilokGeer, do you happen to know why we kept the story private? Should we change the restriction?